packages/amplify-e2e-tests/src/aws-exports/awsExports.ts

```diff
@@ -1,16 +1,25 @@
 import * as fs from 'fs-extra';
 import * as path from 'path';
+import { transformSync } from '@babel/core';
+import babelTransformEsmToCjs from '@babel/plugin-transform-modules-commonjs';
+import { Module } from 'module';
 
 export function getAWSExportsPath(projRoot: string): string {
   return path.join(projRoot, 'src', 'aws-exports.js');
 }
 
 export function getAWSExports(projectRoot: string) {
   const awsExportsPath = getAWSExportsPath(projectRoot);
-  // From Jest 25, ESM modules can only be loaded with mjs extension and Jest takes over
-  // require, that's why we need to copy the file.
-  const awsExportsMJSPath = awsExportsPath.replace('.js', '.mjs');
-  fs.copySync(awsExportsPath, awsExportsMJSPath, { overwrite: true });
-  const localRequire = require('esm')(module);
-  return localRequire(awsExportsMJSPath);
+  const fileContents = fs.readFileSync(awsExportsPath, 'utf-8');
+  // transpile the file contents to CommonJS
+  const { code } = transformSync(fileContents, {
+    plugins: [babelTransformEsmToCjs],
+    configFile: false,
+    babelrc: false,
+  });
+  const mod = new Module('aws-exports.js');
+  // @ts-expect-error This is private api.
+  mod._compile(code, 'aws-exports.js');
+  mod.paths = [projectRoot];
+  return mod.exports;
```

Comment:
The `esm` 3rd party package that was used here before causes V8 to crash with segfault. 
I have replaced this with solution borrowed from here https://github.com/aws-amplify/amplify-cli/blob/2ce4283f7865c225efd95c22e7ad75ae2ce68940/packages/amplify-frontend-javascript/lib/frontend-config-creator.js#L286-L299